apps/antalmanac/src/components/RightPane/SectionTable/CourseInfoButton.tsx

```diff
@@ -65,11 +63,13 @@ function CourseInfoButton({
         }
     };
 
+    const scheduleManagementWidth = useScheduleManagementStore((state) => state.scheduleManagementWidth);
+    const compact = scheduleManagementWidth && scheduleManagementWidth < 600;
```

Comment:
issue: we should keep our breakpoints standard. prefer using a specific breakpoint as defined in our theme (and referencing it as such, not as a magic number)